docs/docs/guides/automate/declarative-automation/index.md

```diff
@@ -1,88 +1,96 @@
 ---
 description: Dagster Declarative Automation is a framework that allows you to access information about events that impact the status of your assets, and the dependencies between them.
 keywords:
-- declarative
-- automation
-- schedule
+  - declarative
+  - automation
+  - schedule
+  - automation condition
 sidebar_position: 20
 title: Declarative Automation
 ---
 
-Declarative Automation is a framework that allows you to access information about events that impact the status of your assets, and the dependencies between them, in order to:
+Declarative Automation is a framework that uses information about the status of your assets and their dependencies to launch executions of your assets.
 
-- Ensure you're working with the most up-to-date data.
-- Optimize resource usage by only materializing assets or executing checks when needed.
-- Precisely define when specific assets should be updated based on the state of other assets.
+Not sure what automation method to use? Check out the [automation overview](/guides/automate) for a comparison of the different automation methods available in Dagster.
 
-Declarative Automation has two components:
+:::note
 
-- An **[automation condition](#automation-conditions)**, set on an asset or asset check, which represents when an individual asset or check should be executed.
-- An **[automation condition sensor](#automation-condition-sensors)**, which evaluates automation conditions and launches runs in response to their statuses.
+In order to enable Declarative Automation, you will need to enable the default **[automation condition sensor](automation-condition-sensors)** in the UI, which evaluates automation conditions and launches runs in response to their statuses.
 
-## Using Declarative Automation
+1. Navigate to **Automation**.
+2. Locate the desired code location.
+3. Toggle on the **default_automation_condition_sensor** sensor.
 
-To use Declarative Automation, you must:
+:::
 
-- [Set automation conditions on assets or asset checks in your code](#setting-automation-conditions-on-assets-and-asset-checks).
-- Enable the automation condition sensor in the Dagster UI:
-  1. Navigate to **Automation**.
-  2. Locate the desired code location.
-  3. Toggle on the **default_automation_condition_sensor** sensor.
-
-## Automation conditions
+## Builtin automation conditions
 
 An <PyObject section="assets" module="dagster" object="AutomationCondition" /> on an asset or asset check describe the conditions under which work should be executed.
 
-Dagster provides a few pre-built automation conditions to handle common use cases:
+The system is flexible, and can be customized to fit your specific needs, but it is recommended to start with one of the built-in conditions below and customize it from there, rather than building up your own condition from scratch.
+
+<Tabs>
+  <TabItem value="on_cron" label="on_cron" default>
```

Comment:
Should we reconsider the ordering of these tabbed options?  Maybe `eager`, then `on_cron`, then `on_missing`?

I also don't know if this tab structure is the best way of presenting this information (as opposed to just having all the different sections inline.

nit: it's weird to see the tab title be `on_cron` vs `AutomationCondition.on_cron(...)`

Reply:
I think `on_cron` should actually be used more often than `eager`, but I'll put `eager` ahead of `on_missing`

Reply:
re: tab structure, I found that without the tabs it was much harder to "feel" when you were starting to read about a new condition type. I also liked that it makes it much easier to compare and contrast different conditions